routes/v1.js

```diff
@@ -163,6 +181,58 @@ api.declare({
   });
 });
 
+
+/** List taskIds by taskGroupId */
+api.declare({
+  method: 'get',
+  route:  '/task-group/:taskGroupId/list',
+  query: {
+    continuationToken: /./,
+    limit: /^[0-9]+$/,
+  },
+  name:   'listTaskGroup',
+  output: 'list-task-group-response.json#',
+  title:  "List Task Group",
+  description: [
+    "List taskIds of all tasks sharing the same `taskGroupId`.",
+    "",
+    "As a task-group main contain an unbounded number of tasks, this end-point",
+    "may return a `continuationToken`. To continue listing tasks you must",
+    "`listTaskGroup` again with the `continuationToken` as the query-string",
+    "option `continuationToken`.",
+    "",
+    "By default this end-point will try to return up to 1000 members in one",
+    "request. But it **may return less**, even if more tasks are available.",
```

Comment:
Because it's one azure call... It could happen in theory.. In practice probably won't... When coding against the API, it's just something worth noting, it won't really complicate peoples code... I hope not... Or if so, they should restructure it.